server/text_generation_server/models/t5.py

```diff
@@ -10,24 +10,18 @@
     AutoModelForSeq2SeqLM,
     AutoConfig,
 )
-from transformers.models.t5.parallel_layers import (
```

Comment:
These are not the same.

Reply:
Oh really ?? How come, what's the difference ?

Those are in your fork right ? Isn't there a way to use the same ones ?

Reply:
https://github.com/OlivierDehaene/transformers/blob/text_generation_inference/src/transformers/models/t5/parallel_layers.py

The difference isn't glaring.

Reply:
Ok, maybe I need to finish the work then. I intended to remove all of these, to get a single loading point, where we don' t have to do post weight passes or after the fact transpositions.

Just load everything directly correctly. But it's starting to become quite a large PR (the full one).

Reply:
I'm fine with large prs ^^

Reply:
I'll still edit that one so that it can pass on its own if you're ok with it.
We still need to move `t5` to custom modeling in order to be on `transformers@main` right ?

Reply:
yes!